src/test/ui/privacy/associated-item-privacy-trait.rs

```diff
@@ -116,15 +115,11 @@ mod priv_parent_substs {
         <Priv as PubTr<_>>::CONST;
         //~^ ERROR type `priv_parent_substs::Priv` is private
 
-        let _: <Pub as PubTr>::AssocTy;
-        //~^ ERROR type `priv_parent_substs::Priv` is private
-        //~| ERROR type `priv_parent_substs::Priv` is private
+        let _: <Pub as PubTr>::AssocTy;  // TODO no longer an error?!
```

Comment:
This also looks extremely wrong to me since the only difference to the code below is the omission of `<_>`, but I can't imagine how this patch caused this, so it might be preexisting?